.github/workflows/test.sh

```diff
@@ -22,12 +22,7 @@ conda activate test-env
 if [ "$USE_CONDA" = "Yes" ]; then
 
     if [ "${1}" = "pyqt5" ]; then
-        if [ "${PYQT5_VERSION:0:4}" = "5.15" ]; then
-            # Must specify versions individually with PyQt 5.15 to avoid compat issues with Python 3.11
-            conda install -q qt-main=${PYQT5_QT_VERSION} qt-webengine=${PYQT5_QT_VERSION} pyqt=${PYQT5_VERSION}
-        else
-            conda install -q qt=${PYQT5_QT_VERSION} pyqt=${PYQT5_VERSION}
-        fi
+        conda install -q qt=${PYQT5_QT_VERSION} pyqt=${PYQT5_VERSION}
```

Comment:
```suggestion
        conda install -q qt-main=${PYQT5_QT_VERSION} qt-webengine=5.15 pyqt=${PYQT5_VERSION}
```
Why not do this?

This is what we do in the qt package. Basically, i couldn't get the version numbers of qt-webengine and qt-main to match up exactly.

Reply:
Because `qt-main` is only available for Qt 5.15.x, and not anything else that we're testing in our matrix (5.9, 5.12, etc). If we do this, we need to have separate logic branches for each version, which is exactly what the script does now, and this PR eliminates.

The goal is to avoid as much fragile, complex, duplicative and non-trivial to maintain version-specific branching logic in this script as possible, and instead keep all of that in the build matrix in the declarative workflow config file (there are still a few bits that should be factored out, but that's a subject for another PR).

Also, I'm not sure why `qt-webengine` is inexplicably hardcoded, instead of using the existing variable, i.e. just using the original line here.

Reply:
Understood